test/transcoding/kernel_arg_type_qual.ll

```diff
@@ -13,10 +13,10 @@ source_filename = "test.cl"
 target datalayout = "e-m:e-p270:32:32-p271:32:32-p272:64:64-i64:64-f80:128-n8:16:32:64-S128"
 target triple = "spir64-unknown-unknown."
 
-; CHECK-SPIRV: String 18 "kernel_arg_type_qual.test.volatile,const,,"
-; CHECK-SPIRV: Name [[ARG:1[0-9]+]] "g"
+; CHECK-SPIRV: String [[#]] "kernel_arg_type_qual.test.volatile,const,,"
```

Comment:
Should we check the uses of these string IDs instead of just ignoring?

Reply:
If I remember correctly - these IDs are unused anywhere in the module. W/A to store argument types qualifiers relies in a kernel name stored in the string to bound it with the actual kernel.

Reply:
Good! Please double-check if they are actually unused, and mark this conversation as resolved

Reply:
Double checked, here is the example (from this test):
  ```
1 119734787 65536 393230 13 0
  2 2 Capability Addresses
  3 2 Capability Kernel
  4 2 Capability Int8
  5 5 ExtInstImport 1 "OpenCL.std"
  6 3 MemoryModel 2 2
  7 5 EntryPoint 6 6 "test"
  8 11 String 11 "kernel_arg_type.test.float*,float*,"
  9 13 String 12 "kernel_arg_type_qual.test.volatile,const,,"
 10 3 Source 3 100000
 11 3 Name 7 "g"
 12 3 Name 8 "c"
 13 3 Name 9 "asd"
 14 4 Name 10 "entry"
 15
 16 3 Decorate 7 Volatile
 17 4 TypeInt 3 8 0
 18 2 TypeVoid 2
 19 4 TypePointer 4 5 3
 20 6 TypeFunction 5 2 4 4 4
 21
 22
 23
 24 5 Function 2 6 0 5
 25 3 FunctionParameter 4 7
 26 3 FunctionParameter 4 8
 27 3 FunctionParameter 4 9
 28
 29 2 Label 10
 30 1 Return
 31
 32 1 FunctionEnd
```

11 and 12 are not used anywhere